app/instance-initializers/ember-a11y.js

```diff
@@ -16,7 +16,13 @@ Ember.Router.reopen({
   }
 });
 
-export function initialize(application) {
+export function initialize(applicationInstance) {
+  if (arguments.length > 1) {
+    applicationInstance = arguments[1];
+  }
+  const container = applicationInstance.container ? applicationInstance.container : applicationInstance.__container__;
```

Comment:
Maybe `container = application ? container || application.container` so we don't have to reach out and snag `__container__`?